docs/source/user/core-structures.rst

```diff
@@ -1,28 +1,98 @@
+.. _core-structures:
+
 Core Structures
 ===============
 
-.. contents:: |toctitle|
-    :local:
-
-Overview
---------
-
-OpenDP is focused on creating computations with specific privacy characteristics. These computations are modeled with two core structures in OpenDP: Transformations and Measurements. These structures are in all OpenDP programs, regardless of the underlying algorithm or definition of privacy. By modeling computations in this abstract way, we're able to combine them in flexible arrangements and reason about the resulting programs.
+OpenDP is focused on creating computations with specific privacy characteristics.
+These computations are modeled with two core structures in OpenDP: Transformations and Measurements.
+These structures are in all OpenDP programs, regardless of the underlying algorithm or definition of privacy.
+By modeling computations in this abstract way, we're able to combine them in flexible arrangements and reason about the resulting programs.
 
 Measurement
 -----------
 
 A Measurement is a randomized mapping from datasets to outputs of an arbitrary type.
+Each Measurement consists of a privacy relation and a function.
```

Comment:
Should we use code styling (rst double-backquotes) for symbols like Measurement? And/or, should we link to the API docs (maybe just on the first usage)?